src/pages/staff/dashboard/MyBrandsPage.jsx

```
@@ -15,16 +15,37 @@ import {
+  // page_view analytics when desktop shell is active
+  useEffect(() => {
+    if (import.meta.env.VITE_EN_DESKTOP_FEED_LAYOUT === 'linkedin' && isDesktop) {
+      try { track('page_view', { page: 'my_brands_desktop', surface: 'community_desktop' }); } catch (err) { console.debug?.('track page_view failed (my_brands_desktop)', err); }
+    }
+  }, [isDesktop]);
   
   // Search state
   const [searchQuery, setSearchQuery] = useState('');
   const [brands, setBrands] = useState([]);
   const [loading, setLoading] = useState(true);
   const [searchLoading, setSearchLoading] = useState(false);
```

ESLint

[error] 'searchLoading' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)